apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -93,7 +147,7 @@ public static BigtableClient Create(ServiceEndpoint endpoint = null, BigtableSer
         public static BigtableClient Create(Channel channel, BigtableServiceApiSettings settings = null, string appProfileId = null)
```

Comment:
I thought we decided this overload was no longer needed.

Reply:
As of right now we need it for Integration tests to create a client to use with the emulator [here](https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/BigtableFixtureBase.cs#L113)
However to make it easier to use Emulator I could add the following method to ClientCreationSettings class
```
/// <summary>
/// Creates <see cref="ClientCreationSettings"/> to be used in creation of <see cref="BigtableClient"/> for use with Emulator.
/// <param name="environmentVariable"> is the environment variable that holds host endpoint values.</param>
/// </summary>
public ClientCreationSettings FromEnvironmentVariable(string environmentVariable)
{
    string endpointString = Environment.GetEnvironmentVariable(environmentVariable);
    string[] endpointValues = endpointString?.Split(':');
    return new ClientCreationSettings(null, null, ChannelCredentials.Insecure, new ServiceEndpoint(endpointValues?[0], Convert.ToInt32(endpointValues?[1])));
}
```
And then in `BigtableFixtureBase.cs` swap
`TableClient = BigtableClient.Create(EmulatorChannel);`
with
`TableClient = BigtableClient.Create(new BigtableClient.ClientCreationSettings().FromEnvironmentVariable(EmulatorEnvironmentVariable));`

Reply:
This will simplify use of the emulator. And if user wants to run some tests against different mock server (like we did) then user will still be able to pass a different custom environment variable to the above method or will have to manually create a `ClientCreationSettings` object with custom settings.